src/main/java/uk/gov/hmcts/reform/userprofileapi/client/UpdateUserProfileData.java

```diff
@@ -18,7 +18,7 @@
     private String idamStatus;
 
     @JsonCreator
-    public UpdateUserProfileData(@JsonProperty(value = "emailAddress") String email,
+    public UpdateUserProfileData(@JsonProperty(value = "email") String email,
```

Comment:
Any reason why this is being changed?

Reply:
This is just to be in sync with Quartz and there was also duplicate field emailAddress was appearing on Swagger

Reply:
Okay great!